### DISCUSSION TITLE
Custom refresh rate

### DISCUSSION THREAD

**Discussion by Samir2zk** [ppy/osu#14290] [Q&A] [answered]
My monitor is a 60hz on and I can't make the game run on my custom refresh rate ,if anyone knows a fix or if it needs to be implemented then please tell me or I really beg for a custom refresh rate support.
(I'm not sure about this but I think it used to work before but I really don't remember)

**osu!lazer version:** 2021.809.0

**Logs:**

**Comment by frenzibyte** [comment, marked as answer]
Are you running in fullscreen mode? sounds like https://github.com/ppy/osu-framework/issues/4081.

**Reply by Samir2zk** [reply]
Yes

**Reply by Samir2zk** [reply]
Thanks for finding me the discussion I was searching for it
